### DISCUSSION TITLE
Audio problems

### DISCUSSION THREAD

**Discussion by LjuandalZPH** [jhassan8/crunchyroll-tizen#96] [Q&A]
Hey everyone i hope you'all are okay.

When I'm watching some anime sometimes the audio goes off for like 20-30 second and then I can hear the audio again.

That happens like 3-4 times per episode, anyone knows how I can fix it?

Thanks

**Comment by jhassan8** [comment]
Hi @LjuandalZPH, does it happen very often? When I started to adapt the app to Tizen I ran into that problem but using the native Tizen player, using HLSJS I had never had that error, a few days ago I played 20 episodes in a row to perform a test and there were no problems, this happens to you at any time. anime or in some specific ones and at specific points in the episode? thanks

**Reply by LjuandalZPH** [reply]
Yes sadly it happens in each episode and anime, hardly ever it doesn't happens.

I download the app with Tizen using developer mode in my tv and with my laptop I did the process

I see animes in a Samsung tv Un43ru7100k

If u want I can record how the problem happens.

Thanks for answer me, have a good one.
